custom_components/tuya_local/devices/catit_pixi_smart_fountain.yaml

```diff
@@ -90,3 +90,46 @@ entities:
         type: integer
         name: sensor
         unit: s
+
+  - entity: sensor
+    name: Water Lack
+    dps:
+      - id: 101
+        type: boolean
+        name: sensor
+
+  - entity: switch
+    name: Eco Mode
+    category: config
+    dps:
+      - id: 102
+        type: integer
+        name: switch
+        mapping:
+          - dps_val: 1
+            value: false
+          - dps_val: 2
+            value: true
+
+  - entity: sensor
+    name: Water State
+    dps:
+      - id: 103
+        type: boolean
+        name: sensor
+
+  - entity: sensor
+    name: Water Empty
```

Comment:
As above, use a binary_sensor with `translation_key: tank_empty`, but since this is the actual empty sensor, remove the name.